next-packages/styled-base/src/index.js

```diff
@@ -29,30 +28,26 @@ let createStyled: CreateStyled = (tag: any, options?: StyledOptions) => {
   }
   let identifierName
   let shouldForwardProp
+  let optionsShouldForwardProp
```

Comment:
I dont like this, but got the expected behaviour in the least amount of code. Would appreciate refactor tips if you have any